### PR TITLE
UI dashboard polish

### PR DESCRIPTION
**Pull request by berggren**
* New sketch summary page
* New create sketch flow
* Fancy event counter

Ref issue: #263 
Reviewer: @someguyiknow

### CODE REVIEW

File: timesketch/ui/static/components/sketch/sketch-count-events-directive.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            },
+            controller: function ($scope) {
+                timesketchApi.countEvents($scope.sketchId).success(function (data) {
+                    $scope.count = data.meta['count'];
+                });
+            }
+        }
+    }]);
+
+})();
```
Comment (someguyiknow) [inline timesketch/ui/static/components/sketch/sketch-count-events-directive.js]: Newline?

File: timesketch/ui/static/components/sketch/sketch-search-template-list.html [hunk 2]
```diff
@@ -12,11 +12,9 @@
...
                         <a href="/sketch/{{ sketchId }}/explore/searchtemplate/{{ template.id }}/">{{ template.name }}</a>
                     </div>
-                    <span style="color:#777;">{{ template.query_string }}</span>
                 </td>
                 <td><button class="btn btn-default" style="margin-top:5px;" ng-click="addSearchTemplate(template)"> <i class="fa fa-plus"></i> Quick add</button></td>
             </tr>
             </tbody>
         </table>
-    </div>
 </div>
```
Comment (someguyiknow) [inline timesketch/ui/static/components/sketch/sketch-search-template-list.html]: Newline?

File: timesketch/ui/static/components/sketch/sketch.js [hunk 3]
```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 (function() {
     var module = angular.module('timesketch.sketch', [
-        'timesketch.sketch.views.directive'
+        'timesketch.sketch.views.directive',
+        'timesketch.sketch.count.events.directive'
     ]);
 })();
```
Comment (someguyiknow) [inline timesketch/ui/static/components/sketch/sketch.js]: Newline?

File: timesketch/ui/templates/sketch/overview.html [hunk 4]
```diff
@@ -259,120 +257,117 @@ <h4 class="modal-title" id="myModalLabel">Delete sketch</h4>
...
+                    </div>
                 </div>
+
             </div>
         </div>
-    </div>
-
     {% endif %}
 
 {% endblock %}
```
Comment (someguyiknow) [inline timesketch/ui/templates/sketch/overview.html]: Newline?

File: timesketch/ui/static/components/story/story-directive.js [hunk 5]
```diff
@@ -34,7 +34,7 @@
                     $scope.stories = [];
                     for (var i = 0; i < stories.length; i++) {
                         var story = stories[i];
-                        story.created_at = moment.utc(story.created_at).format("YYYY-MM-DD HH:MM");
+                        story.created_at = moment.utc(story.created_at).format("YYYY-MM-DD");
```
Comment (someguyiknow) [inline timesketch/ui/static/components/story/story-directive.js]: No hour/min granularity?

Comment (berggren) [inline timesketch/ui/static/components/story/story-directive.js]: oops, fixed

File: timesketch/ui/templates/sketch/overview.html (hunk 4 above)
Comment (berggren) [inline timesketch/ui/templates/sketch/overview.html]: Done

File: timesketch/ui/static/components/sketch/sketch.js (hunk 3 above)
Comment (berggren) [inline timesketch/ui/static/components/sketch/sketch.js]: Done

File: timesketch/ui/static/components/sketch/sketch-search-template-list.html (hunk 2 above)
Comment (berggren) [inline timesketch/ui/static/components/sketch/sketch-search-template-list.html]: Done

File: timesketch/ui/static/components/sketch/sketch-count-events-directive.js (hunk 1 above)
Comment (berggren) [inline timesketch/ui/static/components/sketch/sketch-count-events-directive.js]: Done

**Comment by berggren** [comment]
@someguyiknow PTAL

**Pull request merged**